File: CS/03_Images/AddSVGToPDF.cs
using Spire.Pdf;
using Spire.Pdf.Annotations;
using Spire.Pdf.Graphics;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace AddSVGToPDF
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a new PDF document.
            PdfDocument existingPDF = new PdfDocument();

            //Load an existing PDF
            existingPDF.LoadFromFile(@"..\..\..\..\..\..\Data\SampleB_1.pdf");

            //Create a new PDF document.
            PdfDocument doc = new PdfDocument();

            //Load the SVG file
            doc.LoadFromSvg(@"..\..\..\..\..\..\Data\template.svg");

            //Create template
            PdfTemplate template = doc.Pages[0].CreateTemplate();

            //Draw template on existing PDF
            existingPDF.Pages[0].Canvas.DrawTemplate(doc.Pages[0].CreateTemplate(), new PointF(50, 250), new SizeF(200, 200));
            
            //Save the document
            String result = "AddSVGToPDF_out.pdf";
            existingPDF.SaveToFile(result, FileFormat.PDF);

            //Launch the Pdf file
            PDFDocumentViewer(result);
        }

        private void PDFDocumentViewer(string filename)
        {
            try
            {
                System.Diagnostics.Process.Start(filename);
            }
            catch { }
        }
    }
}

File: VB/03_Images/AddSVGToPDF.vb
Imports Spire.Pdf
Imports Spire.Pdf.Graphics

Namespace AddSVGToPDF
    Partial Public Class Form1
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            ' Create a new PdfDocument object
            Dim existingPDF As New PdfDocument()

            ' Load an existing PDF document from file
            existingPDF.LoadFromFile("..\..\..\..\..\..\Data\SampleB_1.pdf")

            ' Create a new PdfDocument object
            Dim doc As New PdfDocument()

            ' Load an SVG file as a document template
            doc.LoadFromSvg("..\..\..\..\..\..\Data\template.svg")

            ' Create a template from the first page of the document
            Dim template As PdfTemplate = doc.Pages(0).CreateTemplate()

            ' Draw the template onto the canvas of the first page of the existing PDF
            existingPDF.Pages(0).Canvas.DrawTemplate(doc.Pages(0).CreateTemplate(), New PointF(50, 250), New SizeF(200, 200))

            ' Specify the output file name
            Dim result As String = "AddSVGToPDF_out.pdf"

            ' Save the modified existing PDF to a new file
            existingPDF.SaveToFile(result, FileFormat.PDF)

            ' Close the PDF document
            existingPDF.Close()

            ' Launch the Pdf file
            PDFDocumentViewer(result)
        End Sub

        Private Sub PDFDocumentViewer(ByVal filename As String)
            Try
                Process.Start(filename)
            Catch
            End Try
        End Sub
    End Class
End Namespace
